### PR TITLE
(Core/Boss Scripts): Update boss_assembly_of_iron.cpp (Fix: StormcCaller Brundir Fly Die Bug)

### PR DESCRIPTION
**Pull request by ArashGorge**
if Stormcaller Brundir killed in Fly phase him corpse goes to bug and no one cant loot him , with this changes after killing him corpse goes down.
IMG: https://ibb.co/W3dv6mD

##### CHANGES PROPOSED:

-  fixed: teleport corpse to ground 
-  

###### ISSUES ADDRESSED:

Closes 

##### TESTS PERFORMED:

Win 10 Vs 2017 tested in game

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

**Comment by solidmaxtor** [comment]
JOJO you did it, well done Naxx, This script can help other bosses of dungeons that, by the event, can be left in places impossible to loot.

**Comment by ArashGorge** [comment]
> JOJO you did it, well done ArashGorge, This script can help other bosses of dungeons that, by the event, can be left in places impossible to loot.

yes , but i do not remember :D just give name of Dungeons or Boss have bug  :D

File: src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp [hunk 1]
```diff
@@ -12,46 +12,46 @@
 enum AssemblySpells
 {
     // Any boss
-    SPELL_SUPERCHARGE               = 61920,
```
Comment (Stoabrogga) [inline src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp]: Please restore the indentation for all enums to their original state in order to improve readability.

Comment (ArashGorge) [reply src/server/scripts/Northrend/Ulduar/Ulduar/boss_assembly_of_iron.cpp]: edited for improve readability

**Comment by Stoabrogga** [comment]
This PR can be closed in my opinion, as all the changes are also contained in PR #2063.

**Pull request closed without merging**